src/python/serenity_types/pricing/derivatives/options/volsurface.py

```diff
@@ -59,6 +80,11 @@ class VolatilitySurfaceDefinition(BaseModel):
     Strike representation used for this surface, e.g. ABSOLUTE or LOG_MONEYNESS.
     """
 
+    discounting_method: DiscountingMethod
+    """
+    Discounting method used for this surface, e.g. CURVE or FUTURES or DISABLED.
+    """
+
```

Comment:
Should we add discounting_method here?
From code, we are generating different vol_surface interpolated based on different discounting_method

Reply:
Drop it. I believe this is duplicative. It's implied by how the forward was constructed, as you hint at here: if you construct using a pair of DISCOUNTING curves it's DiscountingMethod.CURVE; if it's with a PROJECTION curve it's DiscountingMethod.FUTURES. That said, it might not hurt to reference the curves that went into the vol surface construction in the calibration parameter set, in case someone wanted to reconstruct the surface fitting.

Reply:
What I suggest: push this PR, and I will hit you back with a modification?